ironfish-cli/src/commands/service/bridge/release.ts

```diff
@@ -250,6 +257,63 @@ export default class Release extends IronfishCommand {
     ])
   }
 
+  async processNextMintTransaction(
+    client: RpcSocketClient,
+    account: string,
+    api: WebApi,
+  ): Promise<void> {
+    const nextMintRequests = await api.getBridgeNextMintRequests(1)
+    if (nextMintRequests.length === 0) {
+      this.log('No mint requests')
+      return
+    }
+
+    const mintRequest = nextMintRequests[0]
+
+    const createTransactionResponse = await client.wallet.createTransaction({
+      account,
+      outputs: [
+        {
+          amount: mintRequest.amount,
+          assetId: mintRequest.asset,
+          publicAddress: mintRequest.destination_address,
+          memo: 'Bridged asset',
```

Comment:
I think should this be `mintRequest.id` so that we can update the request status later

Reply:
Ah yeah sorry, I forgot to update this. Fixed